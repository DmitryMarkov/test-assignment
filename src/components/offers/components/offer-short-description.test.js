import React from 'react'
import { render } from '@testing-library/react'
import OfferShortDescription from './offer-short-description'

test('OfferShortDescription should render properly', () => {
  const mockArea = 4242
  const mockBedroomsCount = 2
  const mockGuestsCount = 4
  const { asFragment, getByText } = render(
    <OfferShortDescription
      area={mockArea}
      bedroomsCount={mockBedroomsCount}
      guestsCount={mockGuestsCount}
    />
  )

  expect(getByText(/4242m/i)).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
