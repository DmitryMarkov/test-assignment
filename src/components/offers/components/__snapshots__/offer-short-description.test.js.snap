// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferShortDescription should render properly 1`] = `
<DocumentFragment>
  <div
    class="css-5h0skw"
  >
    4 pers., 2 bedrooms, 4242m
    <sup>
      2
    </sup>
  </div>
</DocumentFragment>
`;
